pure.zsh

```diff
@@ -57,17 +59,28 @@ prompt_pure_clear_screen() {
 }
 
 prompt_pure_check_git_arrows() {
+	# reset git arrows
+	prompt_pure_git_arrows=
```

Comment:
We could actually, but then we'd have to move the arrows icons somewhere else, e.g. this function returns, `0` for no arrows, `1` for right arrow, `2` for left arrow, and `3` for left+right arrow.